SignalServiceKit/src/Storage/YapDatabaseConnection+OWS.m

```diff
@@ -52,6 +52,11 @@ - (nullable NSString *)stringForKey:(NSString *)key inCollection:(NSString *)col
 
 - (BOOL)boolForKey:(NSString *)key inCollection:(NSString *)collection
 {
+    return [self boolForKey:key inCollection:collection defaultValue:NO];
+}
+
+- (BOOL)boolForKey:(NSString *)key inCollection:(NSString *)collection defaultValue:(BOOL)defaultValue
+{
     NSNumber *_Nullable number = [self objectForKey:key inCollection:collection ofExpectedType:[NSNumber class]];
```

Comment:
I think we want something like: `return number ? number.boolValue : defaultValue`

Reply:
Thanks!